cse-valid-adapter-app/src/main/resources/application.yml

```diff
@@ -11,6 +11,8 @@ spring:
           binder: rabbit
           destination: cse-valid-run-task
           group: cse-valid
+        task-status-update:
+          destination: task-manager-status-update
```

Comment:
Please also add that in the configuration :


logging:
  config: /logback-file-location

logback:
  amqp:
    exchange: task-manager-logs-exchange
    
Just so we know it is configurable, and you can add a logback.xml file based on what is done is cse-adapter